src/localization/translations/nl.json

```diff
@@ -93,5 +93,11 @@
     "contact_add_success_toast": "Contact succesvol aangemaakt",
     "contact_add_no_consent_toast": "Zonder bevestiging gaat u niet door met het uitwisselen van inloggegevens en moet u de knop Weigeren gebruiken.",
     "contact_add_cancel_title": "Weet u zeker dat u wilt weigeren?",
-    "contact_add_cancel_message": "Het toevoegen van een contactpersoon is verplicht wanneer u een nieuwe partij tegenkomt in een credential uitwisselingsproces. Houd er rekening mee dat het hele proces voor het uitwisselen van inloggegevens wordt afgebroken als u bevestigt."
+    "contact_add_cancel_message": "Het toevoegen van een contactpersoon is verplicht wanneer u een nieuwe partij tegenkomt in een credential uitwisselingsproces. Houd er rekening mee dat het hele proces voor het uitwisselen van inloggegevens wordt afgebroken als u bevestigt.",
```

Comment:
in een credential uitwisselingsprocess -> bij het uitwissel van credentials

inloggegevens -> gegevens

Reply:
changed